torch/csrc/jit/jit_log.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include <string>
+
+// To enable logging please set(export) PYTORCH_JIT_LOG_LEVEL to
```

Comment:
This comment doesn't seem accurate.
`export PYTORCH_JIT_LOG_LEVEL=JIT_GRAPH_DUMP` is that going to work? I see an `atoi` but no mapping from string to level.